public/app/features/alerting/unified/components/import-to-gma/NamespaceAndGroupFilter.tsx

```diff
@@ -1,9 +1,7 @@
-import { css } from '@emotion/css';
-import { useMemo } from 'react';
+import { useEffect, useMemo } from 'react';
 import { Controller, useFormContext } from 'react-hook-form';
 
-import { GrafanaTheme2, SelectableValue } from '@grafana/data';
-import { Field, Stack, VirtualizedSelect, useStyles2 } from '@grafana/ui';
+import { Combobox, ComboboxOption, Field, Stack } from '@grafana/ui';
```

Comment:
Changed to combobox as select is deprecated, and it does the proper thing with id/label focusing :D